module2/src/com/epam/onedimensionalarrays/ReplaceEverySecondElementByZero.java

```diff
@@ -0,0 +1,10 @@
+package com.epam.onedimensionalarrays;
+
+public class ReplaceEverySecondElementByZero {
+    public static int[] replaceEverySecondElementByZero(int[] array){
+        for (int i = 1; i < array.length; i += 2) {
+            array[i] = 0;
+        }
```

Comment:
Compressing array - it isn't just replacing every 2nd elements by 0, it is like e.g.:
old array - 1 2 3 4 5 6
new array - 1 3 5 0 0 0